### DISCUSSION TITLE
A way to disable graph-related colouring

### DISCUSSION THREAD

**Discussion by q3cpma** [ClementTsang/bottom#1575] [Q&A] [answered]
Hello, is there a way to disable the colours for stuff like CPUs, RAM, network? It's a bit distracting when using the `--basic` mode where these colours don't serve much purpose other than look like a Christmas tree ;)

**Comment by ClementTsang** [comment, marked as answer]
Not as an explicit option, but you can set the colours yourself to just be all the same colour as a workaround, I suppose.

You can also use the `NO_COLOR` env arg to disable color everywhere (e.g. `NO_COLOR=true btm`).

**Reply by q3cpma** [reply]
Hmmm, thanks, I just saw that the config file has much more knobs than the CLI. Sorry for noise, I'll use a specific config for `--basic` and be happy like a clam in high water.
